src/analysis/modifications/loopless.jl

```diff
@@ -11,28 +19,33 @@ Nᵢₙₜ' × G = 0
 yᵢ ∈ {0, 1}
 Gᵢ ∈ ℝ
 i ∈ internal reactions
-Nᵢₙₜ is the nullspace of the internal stoichiometric matrix
+Nᵢₙₜ is the nullspace of the internal stoichiometric matrix (rows =  metabolites, columns = reactions)
 ```
-Note, this modification introduces binary variables, so an optimization solver capable of
-handing mixed integer problems needs to be used. The arguments `max_flux_bound` and
-`strict_inequality_tolerance` implement the "big-M" method of indicator constraints.
+Note, this modification introduces binary variables, so an optimization solver
+capable of handing mixed integer problems needs to be used. The arguments
+`max_flux_bound` and `strict_inequality_tolerance` implement the "big-M" method
+of indicator constraints.
 
-For more details about the algorithm, see `Schellenberger, Lewis, and, Palsson. "Elimination
-of thermodynamically infeasible loops in steady-state metabolic models.", Biophysical
-journal, 2011`.
+For more details about the algorithm, see `Schellenberger, Lewis, and, Palsson.
+"Elimination of thermodynamically infeasible loops in steady-state metabolic
+models.", Biophysical journal, 2011`.
 """
-add_loopless_constraints(;
+add_loopless_constraints(
+    metabolite_idx;
     max_flux_bound = constants.default_reaction_bound, # needs to be an order of magnitude bigger, big M method heuristic
     strict_inequality_tolerance = constants.loopless_strict_inequality_tolerance,
 ) =
     (model, opt_model) -> begin
 
         internal_rxn_idxs = [
-            ridx for (ridx, rid) in enumerate(variables(model)) if
+            ridx for (ridx, rid) in enumerate(reactions(model)) if
             !is_boundary(reaction_stoichiometry(model, rid))
```

Comment:
hm, the reaction for reaction_stoichiometry may not exist here anymore, right? or was the accessor fixed?

Reply:
hmm no, it was not fixed. looking into it